static/app/views/nav/secondary/sections/insights/insightsSecondaryNav.tsx

```diff
@@ -78,102 +73,89 @@ function InsightsSecondaryNav({children}: InsightsNavigationProps) {
     : nonStarredProjects.filter(project => project.isMember).slice(0, 8);
 
   return (
-    <Fragment>
```

Comment:
I recommend viewing the diff without whitespace because I've removed the `Fragment` on each of the secondary nav components. Each of these removes the fragment and no longer accepts `children`